src/test/resources/regressions/features/stubs/stubs0.gobra

```diff
@@ -13,10 +13,10 @@ import (
 )
 
 // Checks that files from package "encoding/binary" are correctly parsed
-func testEncodingBinary(e binary.LittleEndian) {
+func testEncodingBinary() {
 	s := make([]byte, 2)
-	e.PutUint16(s, uint16(0))
-	res := e.String()
+	binary.LittleEndian.PutUint16(s, uint16(0))
```

Comment:
Ah the constants get used as a receiver and you already get the different semantics just because of the type such that the constants' values do not matter

Reply:
Exactly!